Sources/DataPipeline/DataPipelineImplementation.swift

```diff
@@ -196,9 +196,32 @@ class DataPipelineImplementation: DataPipelineInstance {
     }
 
     func trackMetric(deliveryID: String, event: Metric, deviceToken: String) {
```

Comment:
Should we rename it to `trackPushMetric` for clarity? 🤔

Reply:
I am open to ideas. I like the idea of keeping `trackMetric` for backwards compatibility but I also see value in adding another function `trackPushMetric` that could simply perform an internal call to `trackMetric`? 

If you think we should perform a rename or function add, perhaps sending a message to the cdp slack channel? Since it's public api related, I think team input would be valuable.